arrow-fx-coroutines/src/main/kotlin/arrow/fx/coroutines/Platform.kt

```diff
@@ -12,7 +12,7 @@ import kotlin.coroutines.startCoroutine
 const val ArrowExceptionMessage =
   "Arrow-kt internal error. Please let us know and create a ticket at https://github.com/arrow-kt/arrow/issues/new/choose"
 
-internal open class ArrowInternalException(override val message: String = ArrowExceptionMessage) : RuntimeException(message)
+open class ArrowInternalException(override val message: String = ArrowExceptionMessage) : RuntimeException(message)
```

Comment:
I'm not sure how to feel about a globally-available-open arrow internal exception 😕

Reply:
It's used across several modules in Arrow Fx Coroutines, and I didn't see a big problem which such a type being visible for users. I'm not sure which issues this could cause?

Reply:
From an api design perspective the users don't need to be able to instantiate or extend an internal exception. This is normally fixed by having a common internal module with all our internal exceptions, functions, etc. that are only exposed to arrow itself but not outside. Although I understand it's out of the scope of this PR I believe we should discuss (soon) which way we want to go in a ticket/thread

Reply:
Duplicated and made internal.